mlir-sycl/lib/Dialect/IR/SYCLOpsAlias.cpp

```diff
@@ -80,6 +80,12 @@ SYCLOpAsmInterface::getAlias(mlir::Type Type, llvm::raw_ostream &OS) const {
        << mlir::sycl::accessAddressSpaceAsString(Atomic.getAddrSpace());
     return AliasResult::FinalAlias;
   }
+  if (const auto MultiPtr = Type.dyn_cast<mlir::sycl::MultiPtrType>()) {
+    OS << "sycl_multi_ptr_" << MultiPtr.getDataType() << "_"
+       << mlir::sycl::accessAddressSpaceAsString(MultiPtr.getAddrSpace()) << "_"
+       << mlir::sycl::decoratedAddressAsString(MultiPtr.getDecorAddr());
+    return AliasResult::FinalAlias;
+  }
```

Comment:
This needs rebasing (now in `SYCLDialect.cpp`).

Reply:
Done